app/experimenter/experiments/api/v3/serializers.py

```diff
@@ -66,14 +81,46 @@ def create(self, validated_data):
             {
                 "slug": slugify(validated_data["name"]),
                 "owner": self.context["request"].user,
+                "firefox_min_version": Experiment.VERSION_CHOICES[0][0],
```

Comment:
Wouldnt this currently just be hardcoding everything to Firefox `"55.0"`, per app/experimenter/experiments/constants.py#L144?

Reply:
Exactly, it's a hard coded dummy value until we get Firefox versions hooked up, as I called out in the list of todos at the top of the PR 👍